server/cmwell-irw/src/test/scala/cmwell/irw/IRWCassSpec.scala

```diff
@@ -265,18 +265,20 @@ trait IRWCassSpec extends AsyncFlatSpec with Matchers with IRWServiceTest {
     }.toMap
 
     val fatFoton = ObjectInfoton("/irw/xyz/fatfoton1", "dc_test", None, lotsOfFields, None)
-    irw.writeAsync(fatFoton)
-    cmwell.util.concurrent.spinCheck(100.millis, true)(irw.readPathAsync("/irw/xyz/fatfoton1")) {
-      res => res match {
-        case FullBox(readInfoton) => readInfoton == fatFoton
-        case _ => false
+    irw.writeAsync(fatFoton).flatMap(_ => {
```

Comment:
This whole block should be:
```
    irw.writeAsync(fatFoton).flatMap { _ =>
      cmwell.util.concurrent.spinCheck(100.millis, true)(irw.readPathAsync("/irw/xyz/fatfoton1")) {
        case FullBox(readInfoton) => readInfoton == fatFoton
        case _ => false
      }
    }.map { res =>
      withClue(res) {
        case FullBox(readInfoton) => readInfoton shouldBe fatFoton
        case EmptyBox => fail("/irw/xyz/fatfoton1 was not found")
        case BoxedFailure(e) => fail("error occured", e)
      }
    }
```
1. removed redundand braces
2. form multiline function I used {} and not ()

Reply:
for the withClue it's not compiled without the match ...

Reply:
intelijje can compile it but sbt not.I'm not sure why. I see that other tests also define the match keyword.any reason for that?
[error] /home/liel/dev/cm-well/server/cmwell-irw/src/test/scala/cmwell/irw/IRWCassSpec.scala:289:21: missing parameter type for expanded function
[error] The argument types of an anonymous function must be fully known. (SLS 8.5)
[error] Expected type was: org.scalatest.compatible.Assertion
[error]       withClue(res) {
[error]                     ^
[error] one error found
[error] (irw / Test / compileIncremental) Compilation failed